UI/org.eclipse.birt.report.debug.core/src/org/eclipse/birt/report/debug/core/i18n/nls.properties

```diff
@@ -1,12 +1,16 @@
 #/*******************************************************************************
 # * Copyright (c) 2006, 2008 Actuate Corporation.
-# * All rights reserved. This program and the accompanying materials
-# * are made available under the terms of the Eclipse Public License v1.0
-# * which accompanies this distribution, and is available at
-# * http://www.eclipse.org/legal/epl-v10.html
-# *
+# * All rights reserved.
+# * 
+# * This program and the accompanying materials are made available under the
+# * terms of the Eclipse Public License 2.0 which is available at
+# * https://www.eclipse.org/legal/epl-2.0/.
+# * 
+# * SPDX-License-Identifier: EPL-2.0
+# * 
 # * Contributors:
 # *  Actuate Corporation  - initial API and implementation
+# *  Others: See git history
```

Comment:
This should not be added if there is already an existing EPL header. Only add it if the header does not exist. This applies to all entries,